src/main/java/io/dockstore/provision/DOSPluginUtil.java

```diff
@@ -77,8 +77,7 @@
             }
 
         } catch (JSONException e) {
-            System.err.println("getResponse() Error: " + e.getMessage());
-            e.printStackTrace();
+            System.err.println("Error: " + e.getMessage());
```

Comment:
@coverbeck I remember asking this question before, but I don't remember what we decided. 
Do regular log messages work in plugins and if so we should probably add a note to https://github.com/ga4gh/dockstore/tree/develop/dockstore-file-plugin-parent

Reply:
@denis-yuen, we logged an issue for it to investigate, ga4gh/dockstore#1483, but I haven't looked into it.